generators/app/templates/vsts-task/src/helper.ts

```diff
@@ -45,15 +45,15 @@ class Helper {
                 reject(new Error('Invalid params.'));
             }
 
-            request.get(teamProjectCollectionUri, { 'auth': { 'bearer': accessToken}}, 
+            request.get(teamProjectCollectionUri, { 'auth': { 'bearer': accessToken}},
                 (err: any, response: any, data: string) => {
                     if (!err && (response.statusCode === 200)) {
                         try {
-                            resolve(+JSON.parse(data).count);
+                            resolve(+JSON.parse(data).length);
```

Comment:
why did you change this? the API response that displays the count of team project is called count, this isn't looking at an array